binary/_mac/vlc/vlc.py

```diff
@@ -0,0 +1,50 @@
+import shutil
+
+import info
+from Package.BinaryPackageBase import *
+
+
+class subinfo(info.infoclass):
+    vlc_ver = None
+
+    def setTargets(self):
+        for ver in ["2.2.6", "2.2.7", "2.2.8", "3.0.0", "3.0.1", "3.0.2", "3.0.3", "3.0.4", "3.0.5", "3.0.6"]:
+            self.targets[ver] = f"http://download.videolan.org/pub/videolan/vlc/{ver}/macosx/vlc-{ver}.dmg"
+            self.targetInstSrc[ver] = f"vlc-{ver}"
+            self.targetDigestUrls[ver] = f"http://download.videolan.org/pub/videolan/vlc/{ver}/macosx/vlc-{ver}.dmg.sha256"
+            self.patchToApply[ver] = [("vlc-plugin-headers.diff", 1)]
+        self.webpage = "https://www.videolan.org/"
+        self.description = "an open-source multimedia framework"
+
+        self.defaultTarget = "3.0.6"
+
+    def setDependencies(self):
+        self.buildDependencies["virtual/bin-base"] = None
+
+
+class Package(BinaryPackageBase):
+    def __init__(self):
+        BinaryPackageBase.__init__(self)
+
+    def unpack(self):
```

Comment:
Uh just saw the code now, is that really needed?
https://github.com/KDE/craft/blob/master/blueprints/dev-utils/cmake/cmake.py looks less complicated, or is it just for the patching?

Reply:
There is a difference that, in CMake we used, for example, https://cmake.org/files/v3.14/cmake-3.14.0-Darwin-x86_64.tar.gz 

And in this, VLC only provids a .dmg image. So we have to mount the image and extract the file in it by this way.

There is a problem that, in installDir, we have only `include` and `lib`, which are extracted from VLC.app package.

Otherwise, I didn't find a DMGBinaryPackageBase in Craft. If we'd like to get one ?

Reply:
For example, VLC provides a .dmg and its signatures here
http://download.videolan.org/pub/videolan/vlc/3.0.6/macosx/

Reply:
I just added an experimental vlc src package (dosn't build on mac for me yet) maybe we should go that way.

Reply:
No problem, I saw that just now :)

In fact yesterday I tried to build vlc src with a blueprint in Craft. But there is not so much progress.

But build from src is really a better solution